src/admin/content/views/ContentDetail.tsx

```diff
@@ -278,7 +278,6 @@ const ContentDetail: FunctionComponent<ContentDetailProps> = ({ history, match,
 
 							<Table horizontal variant="invisible" className="c-table_detail-page">
 								<tbody>
-									{/*TODO remove cast after typings update to 2.15.0*/}
```

Comment:
moet je de cast dan ook niet wegdoen?
```
src={(contentPage as any).thumbnail_path}
```

Reply:
Hier kan die nog niet weg, nog een foutje in typings.

Reply:
ok, dan mag je de comment niet wegdoen, maar enkel het release nummer updaten :)

Reply:
Correct.